src/hotspot/share/classfile/vmIntrinsics.hpp

```diff
@@ -818,6 +818,11 @@ class methodHandle;
                                        "Ljava/util/function/BiFunction;)Ljava/lang/Object;")                                                   \
    do_name(vector_binary_op_name,     "binaryOp")                                                                                              \
```

Comment:
I think this is not used anymore. I didn't do any cleanup, since currently unary/ternary still use the old version. We can do the cleanup together with unary/ternary in future.